### PR TITLE
add global functions

### PR DESCRIPTION
**Pull request by nolouch**
Signed-off-by: nolouch <nolouch@gmail.com>

### CODE REVIEW

File: log.go [hunk 1]
```diff
@@ -72,16 +76,16 @@ func initFileLog(cfg *FileLogConfig) (*lumberjack.Logger, error) {
...
+	return lg, r
 }
 
 var (
-	_globalMu sync.RWMutex
-	_globalL  = newStdLogger()
-	_globalS  = _globalL.Sugar()
+	_globalMu          sync.RWMutex
+	_globalL, _globalR = newStdLogger()
+	_globalS           = _globalL.Sugar()
```
Comment (rleungx) [inline log.go]: When do we need to use `sugar`?

Comment (nolouch) [reply log.go]: I'm not sure in our situation but keep supporting it.

File: global.go [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+// disabled.
+func Fatal(msg string, fields ...zap.Field) {
+	L().Fatal(msg, fields...)
+}
+
+// SetLevel alters the logging level.
+func SetLevel(l zapcore.Level) {
+	_globalMu.Lock()
+	defer _globalMu.Unlock()
+	_globalR.Level.SetLevel(l)
```
Comment (zimulala) [inline global.go]: Why not use `_globalL `?

**Comment by siddontang** [comment]
Be remember that there is a problem when we use Mutex for GetLevel, see https://github.com/pingcap/tidb/issues/7986

File: global.go [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+
+import (
+	"go.uber.org/zap"
+	"go.uber.org/zap/zapcore"
+)
+
+// Info logs a message at InfoLevel. The message includes any fields passed
+// at the log site, as well as any fields accumulated on the logger.
+func Info(msg string, fields ...zap.Field) {
+	L().Info(msg, fields...)
```
Comment (zimulala) [inline global.go]: If I use this function, it seems the log will print `[INFO] [global.go:24] ...`.

File: global.go [hunk 4]
```diff
@@ -18,30 +18,36 @@ import (
 	"go.uber.org/zap/zapcore"
 )
 
+// Debug logs a message at DebugLevel. The message includes any fields passed
+// at the log site, as well as any fields accumulated on the logger.
+func Debug(msg string, fields ...zap.Field) {
+	L().WithOptions(zap.AddCallerSkip(1)).Debug(msg, fields...)
```
Comment (zimulala) [inline global.go]: Could we add this option to the logger directly when creating the logger?

Comment (nolouch) [reply global.go]: Yes, we can. But I do not directly create the logger with this option because the user may use `L()`,`S()` without the export function.

**Pull request merged**